.jshintrc

```diff
@@ -24,6 +24,7 @@
 		"Promise": true
 	},
 	"strict": false,
+	"esnext": true,
```

Comment:
I'm not sure IE9 supports this.  We should write tests / code so it will run on IE9.  We haven't setup testing for it yet ... but it's coming.